deployment/terraform/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "environment" {
+  description = "Dev/QA/QC/Prd"
```

Comment:
We should probably change it to : "dev,qa,prd,service"

Reply:
Fixed in https://github.com/kids-first/kf-lambda-member-update/pull/15/commits/863c16d1a98cda0f86463dabd971ba715dc8162e